src/py/flwr/common/typing.py

```diff
@@ -258,3 +258,11 @@ class Fab:
 
 class RunNotRunningException(BaseException):
     """Raised when a run is not running."""
+
+
+class AbortRunException(BaseException):
```

Comment:
Should we name this `InvalidRunStatus` or something along those lines? `AbortRun` sounds like the run needs to be aborted, which isn't the case.

Reply:
Changed to InvalidRunStatusException to maintain consistent *Exception naming.